### DISCUSSION TITLE
显示错误 13.3寸超高清接外显示器的情况下

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2165] [Q&A]
4k屏幕 外接显示器是22寸1920的 就是仅这个屏幕（关闭了原屏幕）
然后就是这样的额
好像没有图片上传啊

**Comment by tamlok** [comment]
Hi，

图片直接粘贴就能上传。

**Comment by zhuhaisto** [comment]
![QQ截图20220705040856](https://user-images.githubusercontent.com/108032980/177212626-24643237-9692-4776-8417-8b9db2013433.jpg)
1. 这样设置了后 笔记本外接显示器是能正常使用的 （笔记本超高清屏 外接显示器1920 1080的屏 关闭了一个屏幕的情况下）
2. 但是把外接显示器拔掉 重新启动笔记本 然后会模糊
![QQ截图20220705040955](https://user-images.githubusercontent.com/108032980/177212765-6a02ab98-6775-4955-be8e-eeae3c3d86f6.jpg)

就是这样的模糊 但是如果取消兼容她就正常了
正常是这样子的
![图片](https://user-images.githubusercontent.com/108032980/177212898-f14e720a-1209-4da5-8369-9b7db7a7a789.png)

**Reply by zhuhaisto** [reply]
另外 如果用扩展屏也有点儿问题呢 这算bug吗 大佬能确定吗？

**Reply by tamlok** [reply]
vnote如果在两个屏幕间移动，需要确保dpi一样，不会动态适配。所以你要vnote在哪个屏幕上使用，就要让vnote在哪个屏幕上启动（要排除最小化到系统托盘的情况）。
